contracts/src/core/EigenDAServiceManagerStorage.sol

```diff
@@ -26,25 +26,27 @@ abstract contract EigenDAServiceManagerStorage is IEigenDAServiceManager {
      */
     uint32 public constant BLOCK_STALE_MEASURE = 150;
 
-    /** 
-     * @notice The quorum adversary threshold percentages stored as an ordered bytes array 
-     * this is the percentage of the total stake that must be adversarial to consider a blob invalid
+    /**
+     * @notice The quorum adversary threshold percentages stored as an ordered bytes array
+     * this is the percentage of the total stake that must be adversarial to consider a blob invalid.
+     * The first byte is the threshold for the first quorum, the second byte is the threshold for the second quorum, etc.
```

Comment:
"first quorum" -> "quorum ID 0", this mapping from quorum ID to this array is strict